packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -0,0 +1,92 @@
+    color: var(--color);
+
+    outline: none;
+}
+
+.hop-Popover__dialog {
+    --hop-Popover-background: var(--hop-neutral-surface);
+    --hop-Popover-font-family: var(--hop-body-sm-font-family);
+    --hop-Popover-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-font-weight: var(--hop-body-sm-font-weight);
+    --hop-Popover-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-color: var(--hop-neutral-text);
+    --hop-Popover-title-font-family: var(--hop-body-sm-font-family);
+    --hop-Popover-title-font-weight: var(--hop-body-sm-semibold-font-weight);
+    --hop-Popover-content-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-content-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-content-color: var(--hop-neutral-text-weak);
+    --hop-Popover-footer-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-footer-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-footer-link-color: var(--hop-primary-text);
+    --hop-Popover-border: 0.0625rem solid var(--hop-neutral-border-weak);
+    --hop-Popover-border-radius: var(--hop-shape-rounded-md);
+    --hop-Popover-box-shadow: var(--hop-elevation-lifted);
+    --hop-Popover-max-inline-width: 25rem;
+
+    /* Internal variables */
+    --font-family: var(--hop-Popover-font-family);
+    --font-size: var(--hop-Popover-font-size);
+    --font-weight: var(--hop-Popover-font-weight);
+    --line-height: var(--hop-Popover-line-height);
+
+    display: grid;
+    grid-template-areas:
+        "header header"
+        "content content"
+        "footer action";
+    grid-template-columns: repeat(2, auto);
+    gap: var(--hop-space-inset-sm);
+
+    box-sizing: border-box;
+    inline-size: var(--hop-Popover-max-inline-width);
+    max-inline-size: var(--hop-Popover-max-inline-width);
+    padding: var(--hop-space-inset-md);
+
+    background-color: var(--hop-Popover-background);
+    border: var(--hop-Popover-border);
+    border-radius: var(--hop-Popover-border-radius);
+    box-shadow: var(--hop-Popover-box-shadow);
+}
+
+.hop-Popover__title {
+    --font-family: var(--hop-Popover-title-font-family);
+    --font-weight: var(--hop-Popover-title-font-weight);
+    --color: var(--hop-Popover-color);
+
+    grid-area: header;
+}
+
+.hop-Popover__actions,
+.hop-Popover__action {
+    grid-area: action;
+    justify-self: end;
+    margin-block-start: var(--hop-space-stack-sm);
+}
+
+.hop-Popover__title + :where(p, span, div) {
+    --font-size: var(--hop-Popover-content-font-size);
+    --line-height: var(--hop-Popover-content-line-height);
+    --color: var(--hop-Popover-content-color);
+
+    grid-area: content;
+    margin: 0;
+    padding: 0;
+}
+
+.hop-Popover__footer {
+    --font-size: var(--hop-Popover-footer-font-size);
+    --line-height: var(--hop-Popover-footer-line-height);
+    --color: var(--hop-Popover-color);
+
+    grid-area: footer;
+    place-self: center start;
+}
+
+.hop-Popover__footer :where(a) {
```

Comment:
placeholder note, but this feels odd. Link styling / variant should be passed to through a context

Reply:
We need to define a text color for the popover content, and the way the link component is made, it inherits this color. 

Let me explain, even if we skip the style in the context of the link:
```
[LinkContext, {
        size: "sm",
        isQuiet: true,
        color: "primary"
    }]
]}
```

Once in the DOM, the `a` has the style applied, but the span inside inherits the popover color.

### a
<img width="886" alt="Capture d’écran, le 2024-06-03 à 11 09 37" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/a91fa47f-1e7b-4554-870d-b1e4368efbea">

### span
<img width="1077" alt="Capture d’écran, le 2024-06-03 à 11 10 09" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/dc590274-1018-4167-8671-8633dd1df4bf">

Reply:
The color of the link should most likely be set via a variant, not the Color prop: 

https://6482172cea3f45dd1bb6ef3f-xanhaajuug.chromatic.com/?path=/story/docs-link--variants

Reply:
Even when setting the variant, the color is not applied. 

<img width="426" alt="Capture d’écran, le 2024-06-07 à 16 57 58" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/7c6472bc-73b2-462a-b992-cf51a197e64b">

Reply:
The last commit fixes the problem with the link and text color.

Reply:
is the css still necessary?

Reply:
i'd like to avoid css selector that target unspecified things, such as ` .hop-Popover__footer :where(a)`

we can usually pass classnames to links using the context, so i'd prefer having a selector such as

.hop-Popover__footer .hop-Popover__link

Reply:
The selector was removed in my last commit because it's no longer needed.

Reply:
my bad i looked at an old comment an it changed the files shown in the review to show me the old ones. Closing this comment